internal/setup/setup_test.go

```diff
@@ -1309,6 +1309,28 @@ var setupTests = []setupTest{{
 		`,
 	},
 	// TODO this should be an error because the content does not match.
+}, {
+	summary: "chisel-v1 is deprecated",
+	input: map[string]string{
+		"chisel.yaml": `
+			format: chisel-v1
+			archives:
+				foo:
+					version: 22.04
+					components: [main, universe]
+					suites: [jammy]
+					v1-public-keys: [extra-key]
+					default: true
+			v1-public-keys:
+				extra-key:
+					id: ` + extraTestKey.ID + `
+					armor: |` + "\n" + testutil.PrefixEachLine(testKey.PubKeyArmor, "\t\t\t\t\t\t") + `
+		`,
+		"slices/mydir/mypkg.yaml": `
+			package: mypkg
+		`,
```

Comment:
Nit: Is this needed to run the test? If not, let's remove it.

Reply:
Thought package was needed to run the test properly, but apparently not. Removed it in https://github.com/letFunny/chisel/pull/16/commits/18e61a7dcf9490f62936fa6ae4a864ff68b19a30.